source/docker/wazuh-container.rst

```diff
@@ -99,7 +99,17 @@ Usage
 
       $ git clone https://github.com/wazuh/wazuh-docker.git -b 3.9.0_6.7.1 --single-branch
 
-#. Start Wazuh, Elastic Stack and Nginx using `docker-compose`. From the directory where you have the ``docker-compose.yml`` file:
+2. Start the docker service:
```

Comment:
Please, change to `Docker` with uppercase.

Reply:
Sorry for the delay, it is changed already.